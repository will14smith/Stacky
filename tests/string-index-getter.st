main () -> () { "abc" 1 ! print }