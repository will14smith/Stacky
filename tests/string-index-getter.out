=== Output ===
98

=== Exit ===
0
